extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/__init__.py

```diff
@@ -50,11 +53,12 @@
 
 REMOTE_INFERENCE_CLASSES = [
     "HuggingFaceImage2TextRemoteInference",
+    "HuggingFaceConversationalRemoteInference",
     "HuggingFaceText2ImageRemoteInference",
     "HuggingFaceText2SpeechRemoteInference",
     "HuggingFaceTextGenerationRemoteInference",
     "HuggingFaceTextSummarizationRemoteInference",
     "HuggingFaceTextTranslationRemoteInference",
-    "HuggingFaceAutomaticSpeechRecognitionRemoteInference"
+    "HuggingFaceAutomaticSpeechRecognitionRemoteInference",
```

Comment:
nit: let's keep/move these in alphabetical order

Reply:
updated